src/stringBuffer.js

```diff
@@ -24,7 +24,10 @@
  * @return {function}
  */
 function makeBuffer() {
-  // write code here
+  let result = '';
```

Comment:
result is too general
use `buffer` or `bufferValue` etc.

Reply:
done